src/ja/comico/build.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
     extName = 'Comico'
-    extClass = '.ComicoFactory'
-    extVersionCode = 7
+    extClass = '.Comico'
+    extVersionCode = 8
```

Comment:
since you are changing the package, users will have to uninstall old extension and install new one, so no need to bump version. instead set it to 1